### DISCUSSION TITLE
Support listen systemd.socket

### DISCUSSION THREAD

**Discussion by uasan** [uNetworking/uWebSockets.js#175] [Q&A] [answered]
Hi.

To start the service on demand, support for the `systemd.socket` is necessary, for example, as is done in other servers:
https://github.com/rubenv/node-systemd
```JavaScript
app.listen('systemd', ...);
```
Is planned implement?
Thanks.

**Comment by ghost** [comment, marked as answer]
No that sounds like a bad idea.

**Comment by uasan** [comment]
Ok, maybe zero port (0 `fd` STDIN)?
This is also a working for systemd.socket

```JS
app.listen(0, ...);
```
